packages/itwinui-react/src/core/utils/components/Resizer.tsx

```diff
@@ -216,99 +234,80 @@ export const Resizer = (props: ResizerProps) => {
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          top: 0,
-          left: 0,
-          width: 12,
-          height: 12,
-          cursor: 'nw-resize',
-        }}
+        style={{ cursor: 'nw-resize' }}
       />
       <div
         data-iui-resizer='top'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          top: 0,
-          left: 8,
-          right: 8,
-          height: 8,
-          cursor: 'n-resize',
-        }}
+        style={{ cursor: 'n-resize' }}
       />
       <div
         data-iui-resizer='top-right'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          top: 0,
-          right: 0,
-          width: 12,
-          height: 12,
-          cursor: 'ne-resize',
-        }}
+        style={{ cursor: 'ne-resize' }}
       />
       <div
         data-iui-resizer='right'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          top: 8,
-          right: 0,
-          bottom: 8,
-          width: 8,
-          cursor: 'e-resize',
-        }}
+        style={{ cursor: 'e-resize' }}
       />
       <div
         data-iui-resizer='bottom-right'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          bottom: 0,
-          right: 0,
-          width: 12,
-          height: 12,
-          cursor: 'se-resize',
-        }}
+        style={{ cursor: 'se-resize' }}
       />
       <div
         data-iui-resizer='bottom'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          bottom: 0,
-          left: 8,
-          right: 8,
-          height: 8,
-          cursor: 's-resize',
-        }}
+        style={{ cursor: 's-resize' }}
       />
       <div
         data-iui-resizer='bottom-left'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          bottom: 0,
-          left: 0,
-          width: 12,
-          height: 12,
-          cursor: 'sw-resize',
-        }}
+        style={{ cursor: 'sw-resize' }}
       />
       <div
         data-iui-resizer='left'
         onPointerDown={onResizePointerDown}
-        style={{
-          position: 'absolute',
-          top: 8,
-          left: 0,
-          bottom: 8,
-          width: 8,
-          cursor: 'w-resize',
-        }}
+        style={{ cursor: 'w-resize' }}
       />
     </>
   );
 };
+
+// ----------------------------------------------------------------------------
+
+const ResizerStyles = React.memo(() => <style>{resizerStyles}</style>);
+
+const resizerStyles = /* css */ `
+[data-iui-resizer] {
+  pointer-events: auto;
+  grid-area: 1 / 1 / -1 / -1;
+  width: 12px;
+  height: 12px;
+  z-index: 1;
+}
+[data-iui-resizer='top'],
+[data-iui-resizer='bottom'] {
+  height: 8px;
+  width: auto;
+  z-index: 0;
```

Comment:
Why do the edge resizers get a z-index of 0, while the corners get a z-index of 1?

Reply:
Yes, see PR description

<img width="765" alt="image" src="https://github.com/iTwin/iTwinUI/assets/9084735/ab494225-bdb1-4091-827e-de7e36dde2d4">